### ISSUE TITLE
Add multiple vehicles with different action space

### ISSUE THREAD

**Issue by baimukashev** [Farama-Foundation/HighwayEnv#580]
Hi, I wanted to add multiple vehicles with different action space to the simulator without using multi-agent setting? I want spawn IDM vehicles and add one ego with cont action space for example?

Env seems to have one action space for all vehicles, but is there any ways to  do this? 
self.action_type = action_factory(self, self.config["action"])

Thanks,

**Comment by eleurent** [comment]
Yes it is possible: the `action_type` is only relevant for the controlled vehicle(s) (or ego-vehicle).
Other (uncontrolled) vehicles in the scenes do not follow this action type. In the environment, the `make_vehicles()` method should simply create some `IDMVehicle`s and add them to the scene (which is usually the case by default), and these vehicles will follow the IDM model, while you can still set a continuous action space for the ego-vehicle..
